Add works and about section

kwekubright opened the pull request:
### Description
In the feature i have done the following:

- Added works section to index.html
- Added about section to index.html
- Added flexbox and grid style for works and about section
- Added screenshot of works and icons of JavaScript, HTML and CSS

kwekubright committed:
Update index.html

Add works and about section

kwekubright committed:
Add works screenshots and language icons

Add screenshot images of works and icons of HTML, CSS and JavaScript

kwekubright committed:
Update style.css

Add flexbox and grid styles to works and about section

kwekubright committed:
Update .gitignore

Add images/.DS_Store to gitignore file

kwekubright committed:
Update index.html

Indent <button> in <a> tag for easy reading

kwekubright committed:
Update index.html

Add type attribute to index.html

kwekubright committed:
Update style.css

Update #000 color definition

kwekubright committed:
Update style.css

Reposition selector .work-categories li:not(:first-child) to fix linter error

prolajumokeoni commented on index.html:
```diff
@@ -58,6 +58,161 @@ <h3>Let's connect</h3>
     </section>
 
+    <section class="works">
+      <div class="grid works-grid-container wrap padding-top-0">
```
- [ ] Kindly add all button interactions for the see project  buttons in this section 👍🏿 

![Screenshot_34](https://user-images.githubusercontent.com/69638013/164499753-38ff28c5-22f1-4cda-b0f2-45f1082016aa.png)

prolajumokeoni commented on index.html:
```diff
@@ -58,6 +58,161 @@ <h3>Let's connect</h3>
+            </a>
+          </li>
+        </ul>
+        <button type="button" class="button button-enabled">See my resume</button>
```
- [ ] Kindly add button interactions for the see my resume button in this section 👍🏿 

![Screenshot_34](https://user-images.githubusercontent.com/69638013/164499753-38ff28c5-22f1-4cda-b0f2-45f1082016aa.png)

prolajumokeoni commented on index.html:
```diff
@@ -58,6 +58,161 @@ <h3>Let's connect</h3>
+            <li class="other-info-list">
+              <div class="flex li-flex wrap">
+                <h4>Language</h4>
+                <span> > </span>
```
- [ ] Kindly add the right icon to this header tag,so your design look exactly like the figma design  👍🏿 



Figma Design          |  Your design    
:-------------------------:|:-------------------------:
![Screenshot_35](https://user-images.githubusercontent.com/69638013/164500798-5190cf1a-c05b-4568-af0f-69be1abc34fa.png) | ![Screenshot_33](https://user-images.githubusercontent.com/69638013/164500515-34a1cbd0-17b7-40ff-9e9b-4c25b7be5243.png)

prolajumokeoni reviewed (changes requested):
Hi @kwekubright, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

kwekubright committed:
Add interactions to button element

Add button interaction for hover, focus, disabled

Add the correct icon (down arrow icon) for Languages list

kwekubright committed:
Update style.css

Add new line after "," selector

Danieldotcomcoder reviewed (approved):
Hello @kwekubright, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 👍 👍 😄 👏 

Congratulations! 🎉 🎉 🎉 

![comp-2](https://user-images.githubusercontent.com/87448628/159119684-e86473c4-9b60-43a3-90f2-54048e841394.gif)
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
